puppet/modules/chassis/manifests/hosts.pp

```diff
@@ -3,20 +3,14 @@
 	$aliases = [],
 	$subdomains = false,
 ) {
-	package { [ 'avahi-daemon', 'python-pip', 'python-avahi', 'pkg-config', 'libdbus-glib-1-dev' ]:
+	package { [ 'avahi-daemon', 'python3-pip', 'python-avahi', 'pkg-config', 'libdbus-glib-1-dev' ]:
 		ensure => latest,
 	}
 
-	exec { 'upgrade pip':
-		path    => '/bin:/usr/bin',
-		command => 'pip install --upgrade pip==9.0.3',
-		require => Package['python-pip']
-	}
-
 	ensure_packages( ['mdns-publisher'], {
 		ensure   => present,
-		provider => 'pip',
-		require  => [ Package['python-pip'], Package['libdbus-glib-1-dev'], Exec['upgrade pip'] ],
+		provider => 'pip3',
```

Comment:
Can't think off the top of my head but cold there be any other extensions affected by updating to python 3? I didn't find anything on a search of the chassis org with a provider of 'pip' so I think we're ok.

Reply:
Yeah this is the only python dependency in Chassis and the extensions so I think we should be good. I will do due diligence and test it with the majority of the extensions just to be safe. Thanks for the speedy review 🥰🍆 I didn't want this one sitting in the queue for months like Bionic Beaver!